### PR TITLE
Update micro site with soil colors

### PR DESCRIPTION
**Pull request by caseypt**
Apply the same colors used to symbolize the various soil types on the main site in the micro site. 

**Testing**
- Visit the micro site.
- Select the various soil types and verify that the tile checkmark appears in the correct color for the active soil type.
- Hover over each soil type and verify that the popover header appears in the correct color for the soil type that is being hovered on.

image

Connects to #1123

### CODE REVIEW

File: src/mmw/sass/pages/_water-balance.scss [hunk 1]
```diff
@@ -440,6 +440,38 @@
               background-color: $nlcd-90;
             }
           }
+
+          &.soil-a {
+            border-color: $soil-a;
+
+            &:after {
+              background-color: $soil-a;
```
Comment (rajadain) [inline src/mmw/sass/pages/_water-balance.scss]: The white check-mark is not visible with these light colors:

image

Consider adding `color: #333` in the `&:after` block for all blocks that have dark text over a light background in the pop-up, namely High and Moderate Infiltration in the Soils block, and Wetlands, Grasslands, and Pasture / Hay in the Land Cover block.

**Comment by caseypt** [comment]
Added a commit with your suggestions.

**Comment by rajadain** [comment]
+1 tested! Looks so much more consistent now!

**Comment by caseypt** [comment]
Thanks!

**Pull request merged**